### DISCUSSION TITLE
Getting instance or a new one

### DISCUSSION THREAD

**Discussion by lugosium** [twbs/bootstrap#34254] [Q&A] [answered]
Hi,

Is there a method to quickly get an instance if it exists or create a new one if not?

To avoid this kind of code too often:

`const tooltipInstance = Tooltip.getInstance(this.fileUploadButtonTarget) || new Tooltip(this.fileUploadButtonTarget);`

Thank you

**Comment by GeoSot** [comment, marked as answer]
I think `Tooltip.getOrCreateInstance(element)` is your answer ;)  #33276

It will be launched with 5.0.2

**Reply by lugosium** [reply]
Super! Thank you!
